content/solids/functions.ts

```diff
@@ -148,19 +203,541 @@ export async function templeDisplay2($step: Step) {
+        new Point(-3.6254, -0.6976),
+        new Point(-3.6254, 0.6975),
+        new Point(-4.9525, 1.1287)
+    ),
+    new Polygon( // Curve.020
+        new Point(-5.8708, 7.9091),
+        new Point(-4.8338, 6.9756),
+        new Point(-3.6254, 7.6731),
+        new Point(-3.9155, 9.0378),
+        new Point(-5.3033, 9.1836)
+    ),
+    new Polygon( // Curve.021
+        new Point(-0.1714, 2.5543),
+        new Point(-1.2084, 3.4878),
+        new Point(-2.4169, 2.7902),
+        new Point(-2.1268, 1.4256),
+        new Point(-0.739, 1.2798)
+    ),
+    new Polygon( // Curve.022
+        new Point(3.454, -3.7238),
+        new Point(2.417, -2.7903),
+        new Point(1.2086, -3.4878),
+        new Point(1.4987, -4.8524),
+        new Point(2.8865, -4.9983)
+    ),
+    new Polygon( // Curve.023
+        new Point(3.4539, 0.4616),
+        new Point(2.4169, 1.3951),
+        new Point(1.2085, 0.6975),
+        new Point(1.4986, -0.6671),
+        new Point(2.8863, -0.8129)
+    ),
+    new Polygon( // Curve.024
+        new Point(-2.2454, -1.6311),
+        new Point(-1.2084, -0.6976),
+        new Point(0, -1.3951),
+        new Point(-0.2901, -2.7598),
+        new Point(-1.6779, -2.9056)
+    ),
+    new Polygon( // Curve.025
+        new Point(5.0053, -5.8164),
+        new Point(6.0423, -4.8829),
+        new Point(7.2508, -5.5805),
+        new Point(6.9607, -6.9451),
+        new Point(5.5729, -7.0909)
+    ),
+    new Polygon( // Curve.026
+        new Point(-5.8708, -3.7238),
+        new Point(-4.8338, -2.7903),
+        new Point(-3.6254, -3.4878),
+        new Point(-3.9155, -4.8524),
+        new Point(-5.3033, -4.9983)
+    ),
+    new Polygon( // Curve.027
+        new Point(4.8339, 2.7902),
+        new Point(6.0423, 3.4878),
+        new Point(6.0423, 4.8829),
+        new Point(4.8339, 5.5804),
+        new Point(3.6254, 4.8829),
+        new Point(3.6254, 3.4878)
+    ),
+    new Polygon( // Curve.028
+        new Point(7.2508, -5.5805),
+        new Point(8.4592, -4.8829),
+        new Point(8.4592, -3.4878),
+        new Point(7.2508, -2.7902),
+        new Point(6.0423, -3.4878),
+        new Point(6.0423, -4.8829)
+    ),
+    new Polygon( // Curve.029
+        new Point(8.4592, -7.6732),
+        new Point(9.6676, -6.9757),
+        new Point(9.6676, -5.5805),
+        new Point(8.4592, -4.883),
+        new Point(7.2507, -5.5805),
+        new Point(7.2507, -6.9757)
+    ),
+    new Polygon( // Curve.030
+        new Point(7.2506, -9.7658),
+        new Point(8.4591, -9.0683),
+        new Point(8.4591, -7.6732),
+        new Point(7.2506, -6.9756),
+        new Point(6.0422, -7.6732),
+        new Point(6.0422, -9.0683)
+    ),
+    new Polygon( // Curve.031
+        new Point(3.6253, -7.6732),
+        new Point(4.8338, -6.9756),
+        new Point(4.8338, -5.5805),
+        new Point(3.6253, -4.8829),
+        new Point(2.4169, -5.5805),
+        new Point(2.4169, -6.9756)
+    )
+  ];
```

Comment:
Move this data into a separate file. We already have data for a truncated icosahedron (~= soccer ball) in the shared PolyhedronData file, so not sure whether this is actually needed…